akka-cluster-typed/src/main/scala/akka/cluster/ddata/typed/scaladsl/DistributedData.scala

```diff
@@ -27,25 +25,24 @@ object DistributedData extends ExtensionId[DistributedData] {
  * [[akka.cluster.ddata.DistributedData]] and that means that typed
  * and untyped actors can share the same data.
  */
-class DistributedData(system: ActorSystem[_]) extends Extension {
-  import akka.actor.typed.scaladsl.adapter._
-
-  private val untypedSystem = system.toUntyped.asInstanceOf[ExtendedActorSystem]
-  private val config = system.settings.config.getConfig("akka.cluster.distributed-data")
-  private val settings = ReplicatorSettings(config)
+class DistributedData(system: ActorSystem[_])
+  extends dd.AbstractDistributedData(system.toUntyped.asInstanceOf[ExtendedActorSystem])
```

Comment:
I'm not sure I like that it extends that `dd.AbstractDistributedData`. Is it to reuse some common code?

This means that methods from AbstractDistributedData like `isTerminated` becomes public API although `AbstractDistributedData` itself is `private[akka]`. Can be confusing for us to maintain.

Reply:
‘isTerminated’ is public on untyped Cluster but I think I can make them protected.